src/jit/codegencommon.cpp

```diff
@@ -6640,7 +6641,7 @@ void CodeGen::genMov32RelocatableDataLabel(unsigned value, regNumber reg)
  *
  * Move of relocatable immediate to register
  */
-void CodeGen::genMov32RelocatableImmediate(emitAttr size, unsigned value, regNumber reg)
+void CodeGen::genMov32RelocatableImmediate(emitAttr size, size_t value, regNumber reg)
```

Comment:
Why does this need to be size_t? This function should only handle 32-bit target addresses, not 64-bit host addresses... can it?

Reply:
It actually can handle 64-bit addresses. I can share call stack to show how it happens

Reply:
ok, so we pass the full relocatable target address to emitIns_R_I. Looks like there's a bug between the declaration/definition of this function for arm. Namely, emitarm.h:
```
void emitIns_R_I(instruction ins, emitAttr attr, regNumber reg, ssize_t imm, insFlags flags = INS_FLAGS_DONT_CARE);
```
emitarm.cpp:
```
void emitter::emitIns_R_I(
    instruction ins, emitAttr attr, regNumber reg, int imm, insFlags flags /* = INS_FLAGS_DONT_CARE */)
```
So the implementation takes an `int imm`.

This is going to store a `ssize_t` value.

Reply:
@BruceForstall Corrected `emitIns_R_I` declaration inconsistency in emitarm.cpp. Found more issues like this one with other functions in emitarm.cpp and gonna open another PR to specifically address them.